x/oracle/keeper/oracle.go

```diff
@@ -36,7 +36,13 @@ func (k Keeper) RegisterOracle(ctx sdk.Context, msg *types.MsgRegisterOracle) er
 
 	k.AddOracleRegistrationQueue(ctx, oracleRegistration.UniqueId, oracleAccAddr, oracleRegistration.VotingPeriod.VotingEndTime)
 
-	// TODO: emit RegisterOracle event
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			types.EventTypeOracleRegistration,
+			sdk.NewAttribute(types.AttributeKeyVoteStatus, types.AttributeValueVoteStatusStarted),
+			sdk.NewAttribute(types.AttributeKeyOracleAddress, oracleRegistration.Address),
+		),
+	)
```

Comment:
I've heard that this event emission was already implemented by @inchori in the PR https://github.com/medibloc/panacea-core/pull/393/.

Let's define a ground rule of naming convention.
From [Cosmos SDK doc](https://docs.cosmos.network/v0.46/modules/staking/07_events.html), it would be better to name them as:

### MsgRegisterOracle

|Type|Attr key|Attr val|
|---|---|--|
|register_oracle|oracle_address|panacea1...| 

### EndBlocker

|Type|Attr key|Attr val|
|---|---|--|
|complete_vote|vote_type|oracle_registration or ...| 
|                       |oracle_address|panacea1...|